test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1693,6 +1693,17 @@ def test_softshrink(self):
         x = torch.tensor(-0.5).to(dtype=torch.float32)
         self.run_test(model, x)
 
+    @skipIfUnsupportedMinOpsetVersion(9)
+    def test_atan2(self):
+        class Model(torch.nn.Module):
+            def forward(self, x, y):
+                return torch.atan2(x, y)
+
+        model = Model()
+        x = torch.randn(4, 4)
+        y = torch.randn(4, 4)
+        self.run_test(model, (x, y))
```

Comment:
No need for all these local variables, just in-line:
```suggestion
        self.run_test(Model(), (torch.randn(4, 4), torch.randn(4, 4)))
```